react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+export default class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ({ IMG: 0 });
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(dir) {
+    if (dir === 'left') {
+      if (this.state.IMG === 0) {
+        this.setState({
+          IMG: 4
```

Comment:
What if the `photos` array has more or less than 5 images? How can you use an array to determine its last index?